tools/deployment/linux_packaging/rpm/osqueryd.service

```diff
@@ -13,6 +13,7 @@ ExecStart=/usr/bin/osqueryd \
 Restart=on-failure
 KillMode=control-group
 KillSignal=SIGTERM
+TimeoutStopSec=5
```

Comment:
```suggestion
TimeoutStopSec=15
```

Reply:
@directionless Change made in latest push. Thanks.